src/NuGet.Core/NuGet.PackageManagement/Utility/NetStandardCompatibilityUtil.cs

```diff
@@ -0,0 +1,46 @@
+﻿using NuGet.Frameworks;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace NuGet.PackageManagement
+{
+    internal static class NetStandardCompatibilityUtil
+    {
+        internal static bool DoesProjectTargetsNEt461ORGreater(NuGetFramework currentProjectFramework)
+        {
+            if (currentProjectFramework == null)
+            {
+                throw new ArgumentNullException(nameof(currentProjectFramework));
+            }
+
+            if (currentProjectFramework.IsDesktop()
```

Comment:
[IsDesktop ](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Frameworks/FrameworkExtensions.cs#L21)also returns true for `AspNet ` and `DNX`. Maybe we should check for `NetFramework `explicitly?